source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -140,6 +141,10 @@ def filter_journeys(responses, instance, request):
         FilterMinTransfers(min_nb_transfers=min_nb_transfers),
     ]
 
+    # TODO: we should handle this better....
+    if (request.get('_override_scenario') or instance._scenario_name) == 'distributed':
```

Comment:
Why not enabling it for new_default too?

Reply:
@kinnou02 because the PO said so :D